contrib/google.golang.org/grpc/grpc_test.go

```diff
@@ -30,42 +29,73 @@ func TestClient(t *testing.T) {
 	defer rig.Close()
 	client := rig.client
 
-	span, ctx := tracer.StartSpanFromContext(context.Background(), "a", tracer.ServiceName("b"), tracer.ResourceName("c"))
-
-	resp, err := client.Ping(ctx, &FixtureRequest{Name: "pass"})
-	assert.Nil(err)
-	span.Finish()
-	assert.Equal(resp.Message, "passed")
-
-	spans := mt.FinishedSpans()
-	assert.Len(spans, 3)
+	testcase := map[string]struct {
+		message     string
+		error       bool
+		wantMessage string
+		wantCode    codes.Code
+	}{
+		"OK": {
+			"pass",
```

Comment:
Can you please include the key names? It's not very self-explanatory.

Reply:
You didn't address this comment. Besides adding the keys, would you mind inlining this slice into the `for` loop and getting rid of the variable? That's what we've been doing everywhere:
```go
for name, tt := range []struct{....
```